### DISCUSSION TITLE
Can't install Scrypted via the docs on Ubuntu 22.04.3 LTS

### DISCUSSION THREAD

**Discussion by mrfatboy** [koush/scrypted#1117] [Q&A] [answered]
I'm trying to install scrypted via the docs. 

https://github.com/koush/scrypted/wiki/Installation:-Docker-Compose-Linux

I' getting this error the commands in terminal.

Install Docker? (y/n) y
# Executing docker install script, commit: e5543d473431b782227f8908005543bb4389b8de
+ sh -c apt-get update -qq >/dev/null
E: The repository 'https://ppa.launchpadcontent.net/transmissionbt/ppa/ubuntu jammy Release' does not have a Release file.

Are install scripts not setup for Ubuntu 22.04.3 (Jammy) yet?

Is there something I'm doing wrong or a workaround?

**Comment by koush** [comment]
should work. might need to apt update? not sure. one of the tests is to install on jammy.

**Reply by mrfatboy** [reply]
I ran the command and get this.  I am a newbie with linux.

fatboy@ubuntu-Macmini:~$ sudo apt-get update
Hit:1 https://repo.homebridge.io/ stable InRelease
Hit:2 http://us.archive.ubuntu.com/ubuntu jammy InRelease                                               
Hit:3 http://security.ubuntu.com/ubuntu jammy-security InRelease                                        
Hit:4 http://us.archive.ubuntu.com/ubuntu jammy-updates InRelease                                      
Hit:5 http://us.archive.ubuntu.com/ubuntu jammy-backports InRelease              
Ign:6 https://ppa.launchpadcontent.net/transmissionbt/ppa/ubuntu jammy InRelease
Err:7 https://ppa.launchpadcontent.net/transmissionbt/ppa/ubuntu jammy Release
  404  Not Found [IP: [185.125.190.52](http://185.125.190.52/) 443]
Reading package lists... Done                               
E: The repository 'https://ppa.launchpadcontent.net/transmissionbt/ppa/ubuntu jammy Release' does not have a Release file.
N: Updating from such a repository can't be done securely, and is therefore disabled by default.
N: See apt-secure(8) manpage for repository creation and user configuration details.

**Comment by bjia56** [comment]
`https://ppa.launchpadcontent.net/transmissionbt/ppa/ubuntu` doesn't look like something that the scrypted installer sets up. Was your jammy host previously upgraded in place from an older ubuntu?

**Reply by mrfatboy** [reply]
I installed a fresh copy of Ubuntu 22.04.03 LTS. I did install docker first then found the scrypted instructions.  I then uninstalled Docker and follow the  install docker & scrypted  via the docs and this happened.

**Reply by bjia56** [reply]
What instructions did you use to install docker? I'm guessing the install instructions you used broke your apt settings, so unless you're familiar with how to fix it, it might be better to reinstall Ubuntu and start fresh.

As an aside, it's best to use Docker Compose on Linux (see Scrypted docs for instructions).

**Reply by mrfatboy** [reply]
I followed a youtube video which i can't find now. :(  It was very straight forward.  But then when I went to install scrypted I saw the warning about previously installed docker packages.  So I did a sudo apt-get remove docker -y.  From there the docker-scrypted install scrypt progressed but i now get the error stated.  Is there any way to fix it without starting from scratch?

**Comment by mrfatboy** [comment, marked as answer]
Please close this discussion. It seems a previous install of docker corrupted my system. I started fresh with a new Ubuntu install.
